lib/material/internal/filebrowser/material_filebrowser_internal.flow

```diff
@@ -505,7 +505,19 @@ renderCustomMFileBrowser(manager : MaterialManager, parent : MFocusGroup, captio
 											println
 										);
 										next(closeDialogB, true);
-									}, [MButtonRaised()], [MShortcut("enter"), MEnabled(feq(errorTextT, None()))])
+									}, [], [MEnabled(feq(errorTextT, None()))]),
+									MTextButton(_("CREATE AND OPEN"), \-> {
+										pathName = getValue(state.currentPathB) + getValue(nameB) + "/";
+										state.engine.createDirectory(
+											pathName,
+											\ -> {
+												next(state.currentPathB, pathName);
+												updateFilesViewWithSameScrollPosition();
```

Comment:
```suggestion
												updateFilesViewWithZeroScrollPosition();
```
We don't need to save the window scroll position. Let's start with an empty folder.